src/cmd/add.go

```diff
@@ -26,14 +26,20 @@ var addCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
-		if !template.HasTemplatesDir(appDir) {
-			fmt.Println("no templates found\n\nPlease add templates to the \".exosphere\" folder of your code base.")
-			os.Exit(1)
+		hasTemplates, err := template.HasTemplatesDir(appDir)
+		if err != nil {
+			panic(err)
```

Comment:
Thoughts about using `log.Fatal` for such user errors? Looking at the Go documentation, it seems like `panic` is more for programming errors like accessing a slice index out of bounds or a failing type assertion. Therefore, it prints a stack trace of the issue to stderr, so that the bug in the code can be fixed.

`log.fatal` is for user errors. It logs to the configured log output, without a stack trace which is irrelevant and distracting in this case. 

* https://stackoverflow.com/questions/35996966/when-should-i-use-panic-vs-log-fatalln-in-golang
* https://stackoverflow.com/questions/28472922/when-to-use-os-exit-and-panic-in-golang